src/Altinn.AccessManagement.Core/Services/RegisterService.cs

```diff
@@ -30,5 +29,50 @@ public Task<Party> GetOrganisation(string organisationNumber)
         {
             return _partyClient.LookupPartyBySSNOrOrgNo(organisationNumber);
         }
+
+        /// <inheritdoc/>
+        public Task<Party> GetPartyForPartyId(int partyId, int userId)
+        {
+            return _partyClient.GetPartyAsync(partyId);
+        }
+
+        /// <inheritdoc/>
+        public async Task<Party> GetPartiesForUser(int userId, int partyId)
```

Comment:
Plural name returns single entity. 

I also feel this is not a registry function. This is an authorization/accessmanagement function getting a reportee from the users reporteelist.

Reply:
It should also probably be cached. Consider moving implementation to the ContextRetrievalService maybe?